src/debug/index.js

```diff
@@ -0,0 +1,3 @@
+import { initDevTools } from './devtool';
```

Comment:
We will also need to add this file on the dev build right?

Reply:
Yes I think it should be opt-int by the user if the user adds 

```js
import '../src/debug';
```

they will get support for React dev tool. do you think that there is any other better way of doing this?

Reply:
On rollup we can add this on dev bundle. and exclude it otherwise.